bundles/model/org.eclipse.smarthome.model.thing/META-INF/MANIFEST.MF

```diff
@@ -19,7 +19,8 @@ Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  org.objectweb.asm;bundle-version="[5.0.1,6.0.0)";resolution:=optional,
  org.eclipse.smarthome.model.lazygen;resolution:=optional,
  org.eclipse.xtext.xbase.lib
-Import-Package: org.apache.log4j,
+Import-Package: javax.annotation,

```

Comment:
You should mark it optional, so that it is only required at compile time, but not at runtime.